include/Shark.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include"Animal.h"
+
+class Shark : public Animal
+{
+	void setRandStep();
```

Comment:
i see that this function is a common function  between some classes, so why it's not in the base class ??